foundry.toml

```diff
@@ -6,7 +6,7 @@ cache_path  = 'forge-cache'
 gas_reports = ["*"]
 optimizer = true
 optimizer_runs = 20000
-test = 'test'
+test = 'test/foundry/integration/e2e'
```

Comment:
why this change?

Reply:
so we skip all the other tests now? we probably shouldn't skip all tests.

Reply:
for only run new specific e2e tests for this PR.